api/create.go

```diff
@@ -150,8 +163,8 @@ func CombineTagValidators(validators ...TagValidator) TagValidator {
 	}
 }
 
-func ValidateAndStoreOptionalSystemTag(tagName string, tagValues []string, field **string) error {
-	if err := systemTagValidator(tagName, tagValues); err != nil {
+func ValidateAndStoreOptionalSystemTag(tagName string, tagValues []string, field **string, validator func (string, []string) error) error {
```

Comment:
You can use type `TagValidator` instead instead of the func signature

Reply:
That does seem clearer. I'll sort it out.